aspnetcore/security/data-protection/introTmp.md

```diff
@@ -0,0 +1,113 @@
+---
+title: ASP.NET Core Data Protection Overview
+author: rick-anderson
+description: Learn about the concept of data protection and the design principles of the ASP.NET Core Data Protection APIs.
+ms.author: riande
+ms.custom: mvc
+ms.date: 03/07/2022
+uid: security/data-protection/introTmp
+---
+# DO NOT REVIEW THIS is a temporary file
+
+The ASP.NET Core data protection provides a cryptographic API to protect data, including key management and rotation.
```

Comment:
@tdykstra  this is a temp file I'm using to add the content for https://github.com/dotnet/AspNetCore.Docs/issues/32530